frontend/packages/console-shared/src/components/layout/PageBody.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react';
+import { Flex } from '@patternfly/react-core';
+import * as classNames from 'classnames';
+
+const PageBody: React.FC<PageBodyProps> = ({ children, className, ...props }) => {
+  return (
+    <Flex
+      className={classNames('co-m-page__body', className)}
```

Comment:
I can't find any place where this is used testing or CSS wise

```suggestion

```

Reply:
It's being [used in QE's tests](https://github.com/search?q=org%3Aopenshift%20co-m-page__body&type=code).  Didn't want to break them.